tools/make_and_install.sh

```diff
@@ -0,0 +1,42 @@
+#!/bin/bash
+
+set -ex
+
+mkdir -p 'build' && cd 'build'
+
+if test -z "${OSX_VERSION}"
+then
+  IPO="ON"
+else
+  # Disable IPO for OSX due to unexplained linker failure.
+  IPO="OFF"
+fi
+
+# if test -z "${INSTALL_PREFIX}"
+# then
+#   INSTALL_PREFIX='install'
+# fi
```

Comment:
commented code

Reply:
What happens with `cmake` when you set the `CMAKE_INSTALL_PREFIX` with an empty string?
`-DCMAKE_INSTALL_PREFIX=""`